explorer/src/pages/Overview/index.tsx

```diff
@@ -85,6 +85,9 @@ export const PageOverview: React.FC = () => {
           <Grid item xs={12}>
             <ContentCard
               title={`Current block height is ${formatNumber(block?.data)}`}
+              onClick={() =>
+                window.open('https://testnet-milhon-blocks.nymtech.net/blocks')
+              }
```

Comment:
A hyperlink seems more accessible to me, because:
- users can copy the link
- they can preview where they are going by hovering over it before clicking it

```suggestion
              title={<a href="https://testnet-milhon-blocks.nymtech.net/blocks" target="_blank">Current block height is ${formatNumber(block?.data)}</a>}
```

Reply:
Also `https://testnet-milhon-blocks.nymtech.net` should be a constant

Reply:
Updated to hyperlink and moved URL out to a constant.